yamllint.conf

```diff
@@ -6,8 +6,9 @@ yaml-files:
 
 ignore: |
   .terraform/
-  build/azDevOps/azure/air-infrastructure-data-vars-template.yml
-  build/azDevOps/azure/network/air-infrastructure-data-network-vars-template.yml
+  /app/de_workloads/data_processing/silver/repo*
+  /app/de_workloads/ingest/Ingest_AzureSql_Example/repo*
+  /app/de_workloads/shared_resources/repo*
```

Comment:
@balpurewal It would be better to have these as wildcards (e.g. `/app/de_workloads/*/repo*` (although test that works))